apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -46,6 +51,7 @@ function render_fallback_coming_soon_page() {
 	add_filter( 'jetpack_enable_opengraph', '__return_false', 1 );
 	add_filter( 'wpcom_disable_logged_out_follow', '__return_true', 10, 1 ); // Disable follow actionbar.
 	add_filter( 'wpl_is_enabled_sitewide', '__return_false', 10, 1 ); // Disable likes.
+	add_filter( 'jetpack_implode_frontend_css', '__return_false', 99 ); // Jetpack "implodes" all registered CSS files into one file.
```

Comment:
I'm still a bit unclear on what this is doing. Does it mean we'll ship less CSS because Jetpack won't bundle a whole bunch of unused CSS together?

Reply:
I added it here because wpcomsh thought it prudent to add it to its coming soon site. 

Just ticking boxes here. Happy to take it out.

Reply:
I see, so copy pasting from people who probably knew more than us? I'm down with that :D

Reply:
copy pasting is my life